packages/headless/src/composables/comboBox/createComboBox.ts

```diff
@@ -173,6 +173,11 @@ export const createComboBox = createBuilder(
     };
 
     const handleKeydown = (event: KeyboardEvent) => {
+      if (event.key === "Enter") {
+        // prevent submitting the form when pressing enter
+        event.preventDefault();
+      }
```

Comment:
maybe we should only do this while the flyout is open?

because if you hit enter when it's closed, it might be fine to actually send a form.

also
```suggestion
      if (event.key === "Enter") {
        // prevent submitting on pressing enter when the combo box is inside a form
        event.preventDefault();
      }
```

Reply:
The select has to be focussed in order for the enter key event to fire.
But if the select is focussed, enter should always either open the flyout or select a value if I understood it correctly, right @JoCa96 ?

Reply:
ah, if enter also opens the flyout then yes, it should always be prevented. thx!

Reply:
yep 👍
![image](https://github.com/SchwarzIT/onyx/assets/151019977/ab4944e5-16a7-4708-8149-0ea262774789)